### ISSUE TITLE
Cannot find DenseSolver and some methods in new version (1.7)

### ISSUE THREAD

**Issue by gvoigt** [GreenDelta/openlca-python-tutorial#17]
With version 1.63, after appending the library path to the system, I could import the `DenseSolver` via
```python
from org.openlca.eigen.solvers import DenseSolver
```
As of version 1.7.0 there is no such library as `eigen` anymore. How do I get the `DenseSolver` now?

Also there seems to be some renaming to have a occured. For example `Daos.createBaseDao` changed to `Daos.base`. Am I right? Is there a documentation on how methods where renamed?

**Comment by msrocka** [comment]
Right, the DenseSolver moved to the `org.openlca.core.matrix.solvers` package (see also #15).

There is currently no specific documentation for these changes; just the [commit log](https://github.com/GreenDelta/olca-modules/commits/master)

We could generate the Javadoc for the API but this is probably for most uers of the Python API not so readable (I think even the source code on the Github repo is more readable than what Javadoc is generating (e.g. with the [Octotree extension](https://addons.mozilla.org/de/firefox/addon/octotree/?src=search))).
